app/com/linkedin/drelephant/spark/heuristics/ConfigurationHeuristic.scala

```diff
@@ -77,6 +74,14 @@ class ConfigurationHeuristic(private val heuristicConfigurationData: HeuristicCo
       new HeuristicResultDetails(
```

Comment:
For all configs that we are checking, we should add a detail like of key-value pair like

<configName>   <Expected to be within x-y. but found z instead>

whenever we find that check not yielding none.
thanks
shankar